internal/local/command_kafka_topic.go

```diff
@@ -61,3 +62,8 @@ func initKafkaRest(c *pcmd.CLICommand, cmd *cobra.Command) (*kafkarestv3.APIClie
 
 	return kafkaRestClient, clusterListData.Data[0].ClusterId, nil
 }
+
+func (c *command) getPlaintextBootstrapServers() string {
+	servers := strings.Join(c.Config.LocalPorts.PlaintextPorts, ",:")
+	return ":" + servers
```

Comment:
Can we loop through the array and add ":" to each of the elements, and then use `Join()`? This solution works, but seems really hacky.